lib/helpers/jwt.js

```diff
@@ -108,7 +108,7 @@ class JWT {
     try {
       verified = await createVerify(keyOrStore).verify(jwt);
     } catch (err) {
-      if (isKeyStore(keyOrStore) && keyOrStore.stale && keyOrStore.stale()) {
+      if (isKeyStore(keyOrStore)) {
```

Comment:
I think we at least need a check for the `refresh` function being present on the keyOrStore instance. While i dislike overloading with the node-jose prototype i don't have the time to refactor this to use an internal representation of a keystore :(